.travis.yml

```diff
@@ -1,50 +1,26 @@
-sudo: required
-services:
-  - docker
-language: go
-dist: xenial
-go_import_path: github.com/CanonicalLtd/serial-vault
-go:
-  - 1.13
-env:
-  matrix:
-    - TEST_SUITE="--static"
-    - TEST_SUITE="--unit"
-
-before_install:
-  - go get github.com/cheggaaa/pb
-  - go get gopkg.in/macaroon.v1
-  - go get gopkg.in/retry.v1
-  - go get gopkg.in/tomb.v2
-  - go get gopkg.in/check.v1
-  - go get -u golang.org/x/lint/golint
-  - go get github.com/client9/misspell/cmd/misspell
-  - go get -u github.com/gordonklaus/ineffassign
-  - go get -u github.com/alexkohler/nakedret
-  - go get ./...
-
-install:
-  - echo $GOPATH
-  - echo "Remaining install is done by the test script."
-  - true
-
-script: sh -v ./run-checks $TEST_SUITE
+dist: focal
```

Comment:
Note that it'd be best to test on the same distro where the service is deployed. That said, it's OK to leave this as focal and we should instead change the build/deploy process to use focal :) (tl;dr no change needed here).

Reply:
Yes, that makes sense. I will change the build/test process to use focal in the next PR